source/pkcore/faq.rst

```diff
@@ -9,6 +9,12 @@ What Spigot version does the current plugin version support?
 
 ProjectKorra Core **v1.8.8** is compatible with Spigot **1.13.2** at the moment.
 
+If it still is not operating properly, consider that your Spigot version may not be fully up to date. Spigot **1.13.2** alone has over **400** updates to it. You can check the status of your version by executing the ``/version`` command while on your server. If you are not up to date, try getting an updated Spigot ``.jar`` file. You can achieve this by building the latest version with Build Tools.
+
+Build Tools: https://hub.spigotmc.org/jenkins/job/BuildTools/
```

Comment:
Please see the contributing guidelines for how to properly add/insert links. While having the direct URL will work, we have guidelines for in lining the links so that they can be referenced from anywhere rather than copy pasting it countless times.